src/ServiceInsight/SequenceDiagram/ModelCreator.cs

```diff
@@ -100,6 +102,17 @@ void Initialize()
             }
         }
 
+        private void OrderHandlers()
+        {
+            var start = handlers.Where(h => h.ID == ConversationStartHandlerName);
```

Comment:
Why do we have to special case the conversation start? Is this because `HandledAt` is not granular enough and the conversation start can have the same value as the next handler?